app/views/vacancies/_show.json.jbuilder

```diff
@@ -21,22 +23,29 @@ json.jobLocation do
     json.postalCode vacancy.school.postcode
   end
 end
-json.responsibilities vacancy.job_description
 
-json.title vacancy.job_title
 json.url vacancy_url(vacancy)
 
 json.baseSalary do
   json.set! '@type', 'MonetaryAmount'
-  json.minValue vacancy.minimum_salary
-  json.maxValue vacancy.maximum_salary
   json.currency 'GBP'
+  json.value do
+    json.set! '@type', 'QuantitativeValue'
+    if vacancy.minimum_salary && vacancy.maximum_salary.blank?
+      json.value vacancy.minimum_salary
+    else
+      json.minValue vacancy.minimum_salary
+      json.maxValue vacancy.maximum_salary
+    end
+    json.unitText 'YEAR'
+  end
 end
 
 json.hiringOrganization do
```

Comment:
I can't see that we currently care about `alumni` data but did you see there's an EducationalOrganisation http://schema.org/EducationalOrganization ?